File: LeetCode Solutions/Learn/Arrays 101/Remove Element/Remove Element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        begin = 0
        end = len(nums) - 1
        while (begin <= end):
            if (nums[begin] == val):
                nums[begin] = nums[end]
                end -= 1
            else:
                begin += 1
        return begin

File: LeetCode Solutions/Learn/Array and String/Remove Element/Remove Element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int begin = 0, end = nums.size() - 1;
        while (begin <= end){
            if (nums[begin] == val){
                nums[begin] = nums[end];
                end--;
            } else{
                begin++;
            }
        }
        return begin;
    }
};